README.rst

```diff
@@ -27,15 +27,29 @@ And an Array can be converted to a numpy ndarray on demand::
     array([ 0.72151309,  0.54654914], dtype=float32)
 
 
-Get in touch!
--------------
+------------------------------------------
+Important Note : Future development status
```

Comment:
I wonder if we don't need to preface this heading with "Important note"? I certainly get the sentiment but I wonder whether the content as it is draws reader attention to the fact this is important!

Alternatively, we could stick this block within a note or warning admonition :wink: